sdk/tests/js/webgl-test-utils.js

```diff
@@ -3183,13 +3183,16 @@ var getBaseDomain = function(host) {
 }
 
 var runningOnLocalhost = function() {
-  return window.location.hostname.indexOf("localhost") != -1 ||
-      window.location.hostname.indexOf("127.0.0.1") != -1;
+  let hostname = window.location.hostname;
+  return hostname.indexOf("localhost") != -1 ||
+    hostname.indexOf("127.0.0.1") != -1 ||
+    hostname.indexOf("::1") != -1;
```

Comment:
not sure indexOf is technically the right choice here; why not just use `===` for all of these? I doubt it would come up, but you wouldn't want to match `xxxx:xxxx:xxxx::1`.

Reply:
Good point - not sure why the code was written this way originally. Done.